.gitignore

```diff
@@ -1,3 +1,4 @@
+coverage/
```

Comment:
This is created when running the karma tests (same as under grunt) - I'll add a little note in this file saying so.